reinforcement_learning/start.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+# Copyright 2018 Google Inc. All Rights Reserved.
```

Comment:
`# Copyright 2018 Google LLC`

Reply:
fixed in 5f6279d772a047cdd88035ac4212819b90aee740